src/class-elements/grammar-privatename-in-computed-property-missing.case

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Caio Lima (Igalia SL). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Use of udeclared PrivateName in ComputedProperty is a syntax error
+info: |
+  ClassTail : ClassHeritage { ClassBody }
+    1. Let lex be the LexicalEnvironment of the running execution context.
```

Comment:
This info is using a Syntax template, therefore it would be nice to use the matching grammar:

```
ClassElementName:
  PropertyName
  PrivateIdentifier

PropertyName:
  LiteralPropertyName
  ComputedPropertyName

ComputedPropertyName:
  [ AssignmentExpression ]

AssignmentExpression ... MemberExpression

MemberExpression:
  MemberExpression . PrivateName
```

It also needs to specify where the invalid syntax is happening then, once the current text is showing part of the [Evaluation phase](https://tc39.es/proposal-class-fields/#runtime-semantics-class-definition-evaluation).